### PR TITLE
Added new wiki: importing KVM based VMs to oVirt

### PR DESCRIPTION
**Pull request by shaharha**
Signed-off-by: Shahar Havivi <shaharh@redhat.com>

Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/develop/release-management/features/virt/KvmToOvirt.html.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+feature_name: 'v2v: Importing Libvirt Kvm Vms to oVirt'
+feature_modules: all
+feature_status: Released in oVirt 4.1
+---
+
+# Importing Libvirt KVM VMs to oVirt
+
+## Summary
+Import KVM based VMs from Libvirt management via the oVirt admin portal.
+There is no need to disk conversion since both oVirt and Libvirt support KVM based VMs.
```
Comment (mykaul) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: to disk conversion -> for disk conversion

File: source/develop/release-management/features/virt/KvmToOvirt.html.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Importing VMs with file and block device disks are supported - snapshots are currently not supported but please note that if you want to import VM with snapshots the current snapshot is the one that will be       imported.
```
Comment (mykaul) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: Extra space before the last word.

File: source/develop/release-management/features/virt/KvmToOvirt.html.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+The import process is done via the utility kvm2ovirt, located at '/usr/libexec/vdsm/kvm2ovirt' (kvm2ovirt is a tool that come with VDSM installation).
+Please note that the VMs that you want to import must be in status Down at Libvirt management.
+
+## Libvirt Configuration
+In order to import VM from Libvirt, Libvirt needs to listen on a public port or via ssh protocol.
```
Comment (mykaul) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: or via -> via the
ssh -> SSH

File: source/develop/release-management/features/virt/KvmToOvirt.html.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+ssh protocol uses the 'qemu+ssh://username@host1.example.org/system' and tcp uses 'qemu+tcp://username@host1.example.org/system'
+- to setup via ssh please follow the following link: [ssh-setup](https://wiki.libvirt.org/page/SSHSetup)
+- to setup via tcp please follow the following link: [tcp-setup](http://wiki.libvirt.org/page/Libvirt_daemon_is_not_listening_on_tcp_ports_although_configured_to)
+
+## Importing VM
+Importing VM is done by the VM uniq Libvirt name that can identify via:
```
Comment (mykaul) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: Importing VM -> Importing a VM

uniq -> unique 
that can identify -> that can be identified

File: source/develop/release-management/features/virt/KvmToOvirt.html.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+$ virsh -c qemu+tcp://username@host1.example.org/system
```
Comment (mykaul) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: perhaps '-r' ?

File: source/develop/release-management/features/virt/KvmToOvirt.html.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+$ virsh # list --all
+For example lets say we want to import VM named 'rhel' from 'qemu+tcp://username@host1.example.org/system'
```
Comment (mykaul) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: lets say -> if
VM -> a VM

File: source/develop/release-management/features/virt/KvmToOvirt.html.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+- Login to the admin portal and navigate to the VMs tab
+- Press the 'Import' button on the toolbar
+- in the Source box select 'KVM (via libvirt)' option
+- enter 'qemu+tcp://username@host1.example.org/system' in the URI box
+- press the 'Requires Authentication' checkbox if authentication needed and enter username/password
```
Comment (mykaul) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: needed -> is needed

File: source/develop/release-management/features/virt/KvmToOvirt.html.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+ssh protocol uses the 'qemu+ssh://username@host1.example.org/system' and tcp uses 'qemu+tcp://username@host1.example.org/system'
+- to setup via ssh please follow the following link: [ssh-setup](https://wiki.libvirt.org/page/SSHSetup)
+- to setup via tcp please follow the following link: [tcp-setup](http://wiki.libvirt.org/page/Libvirt_daemon_is_not_listening_on_tcp_ports_although_configured_to)
+
+## Importing VM
+Importing VM is done by the VM uniq Libvirt name that can identify via:
+$ virsh -c qemu+tcp://username@host1.example.org/system
+$ virsh # list --all
+For example lets say we want to import VM named 'rhel' from 'qemu+tcp://username@host1.example.org/system'
+- Login to the admin portal and navigate to the VMs tab
```
Comment (mykaul) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: Missing capital letters for some of the steps here.

File: source/develop/release-management/features/virt/KvmToOvirt.html.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+- Press the 'Import' button on the toolbar
+- in the Source box select 'KVM (via libvirt)' option
+- enter 'qemu+tcp://username@host1.example.org/system' in the URI box
+- press the 'Requires Authentication' checkbox if authentication needed and enter username/password
+- press the 'Load' button
+- in the box 'Virtual Machines on Source' you should see all the VMs that are down
```
Comment (mykaul) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: down -> not running.

Worth mentioning somewhere that you can import VMs only when they are not running?

File: source/develop/release-management/features/virt/KvmToOvirt.html.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+- select the VM 'rhel' and press the right arrow -> to move it to the 'Virtual Machines to Import' box
+- Press the 'Next' button
+- In this dialog you can adjust the VM properties such as 'Operating System' and 'Allocation Policy'
+- Press the 'OK' button
+- You should see the 'rhel' VM listed in the VMs tab and the progress should start shortly
```
Comment (mykaul) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: Screenshots would be nice here.

File: source/develop/release-management/features/virt/KvmToOvirt.html.md (hunk 9 above)
Comment (shaharha) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: It needed to be explicitly down, not running can be Migrating,
maybe 'down status'?

File: source/develop/release-management/features/virt/KvmToOvirt.html.md (hunk 10 above)
Comment (shaharha) [inline source/develop/release-management/features/virt/KvmToOvirt.html.md]: sure

**Comment by mykaul** [comment]
@JohnMarksRH - please review.

**Comment by shaharha** [comment]
will send a new version with images soon

**Pull request closed without merging**